proto/p2p/p2p.proto

```diff
@@ -116,6 +116,8 @@ message Version {
   // Subnets the peer is tracking
   repeated bytes tracked_subnets = 8;
   Client client = 9;
+  repeated uint32 supported_acps = 10;
```

Comment:
torn on whether or not `uint16` or `uint32` here...this message is only sent once per connection right (if so, then `uint32` is ok IMO)?

Reply:
They are encoded as the same size

Reply:
> this message is only sent once per connection right
Yeah we only send this once. Theoretically a peer can send it more than once and we won't close but that's unusual behavior